docs/logs/customizing-the-sdk/README.md

```diff
@@ -30,7 +30,67 @@ TODO
 
 ### SetResourceBuilder
 
-TODO
+[Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
+is the immutable representation of the entity producing the telemetry.
+If no `Resource` is explicitly configured, the default is to use a resource
+indicating this [Telemetry
+SDK](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk).
+The `SetResourceBuilder` method can be used to set a single `ResourceBuilder`.
+If `SetResourceBuilder` is called multiple times, only the last is kept..
+It is not possible to change the resource builder *after* creating the
+`LoggerFactory`.
+
+The snippet below shows configuring a custom `ResourceBuilder` to the provider.
+
+```csharp
+using var loggerFactory = LoggerFactory.Create(builder =>
+{
+    builder
+        .AddOpenTelemetry(options =>
+        {
+            options.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(
+                serviceName: "MyService",
+                serviceVersion: "1.0.0"
+                ));
+            options.AddConsoleExporter();
+        });
+});
+
+var logger = loggerFactory.CreateLogger<Program>();
+
+logger.LogInformation("Hello Information");
+```
+
+This example generates the following output which includes the Resource
```

Comment:
perhaps not needed to show the actual output here. By this time, users already got started with getting-started, so they should be able to run it and see output themselves.

Reply:
I'm a fan of including the output. 
I think demonstrating what these settings affect is valuable.

What if we remove the top half of the example and only include the bottom section?
```
Resource associated with LogRecord:
    service.name: MyService
    service.version: 1.0.0
    service.instance.id: 00000000-0000-0000-0000-000000000000
```

Reply:
I also agree that these are very simple examples and someone should be able to follow along with ease.

Reply:
I'd suggest to remove actual output. We don't do that anywhere else (except in the getting-started guide). By completing getting-started, we expect user should already know where to look for output.